src/dev/impl/DevToys/DevToys.csproj

```diff
@@ -198,6 +203,7 @@
     <Compile Include="Core\Clipboard.cs" />
     <Compile Include="Core\Collections\OrderedObservableCollection.cs" />
     <Compile Include="Core\Constants.cs" />
+    <Compile Include="Core\Formatter\BaseNumberFormatter.cs" />
```

Comment:
I don't remember what you said about it. Do you think we will reuse the base number formatter in another tool? If not, should we move this to a tool-specific folder?